views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -72,6 +72,8 @@ define([
                 text : __('Preview'),
                 cssClass: 'preview-trigger'
             })).on('click', function previewHandler(e){
+                $(document).trigger('open-preview');
```

Comment:
`$(document).trigger` is not an ideal practice.

However I didn't find a better way to talk to the widget from the preview.

You should at least namespace this event (`open-preview.qti-item`) so that the listener wouldn't be triggered by some unrelated code in the future.